crates/languages/src/go/config.toml

```diff
@@ -12,5 +12,3 @@ brackets = [
     { start = "`", end = "`", close = true, newline = false, not_in = ["comment", "string"] },
     { start = "/*", end = " */", close = true, newline = false, not_in = ["comment", "string"] },
 ]
-tab_size = 4
-hard_tabs = true
```

Comment:
This change doesn’t look correct, as Go uses hard tabs.

Reply:
One more small change here, we need to return that trailing newline back.